docs/2.6.5/docs/index.rst

```diff
@@ -0,0 +1,2 @@
+Daml Documentation
+------------------
```

Comment:
This fixes the title of the "root" page to be `Daml Documentation` instead of the current `Daml Files` (which comes from the daml repo bundle's fake index page).